contrib/nydus-snapshotter/Makefile

```diff
@@ -21,19 +21,14 @@ clear:
 	rm -f bin/*
 	rm -rf _out
 
-.PHONY: vet
-vet:
+.PHONY: test
+test: build
 	go vet $(PACKAGES)
-
-check: vet
 	golangci-lint run
-
-.PHONY: test
-test: vet
-	go test -v -mod=mod -cover ${PACKAGES}
+	go test -v -cover ${PACKAGES}
```

Comment:
With this patch, it's no longer easy for us to just perform static check

Reply:
`make test` is your friend ;)